EKF/control.cpp

```diff
@@ -663,27 +663,24 @@ void Ekf::controlGpsFusion()
 
 		// handle the case when we now have GPS, but have not been using it for an extended period
 		if (_control_status.flags.gps) {
-			// Handle divergence shortly after takeoff casued by a bad yaw angle. Be faster to reset yaw
```

Comment:
Do we still want to make a distinction  between the post takeoff and other phases of flight? I guess it is application specific and if the EKF-GSF is reliable during up and away flight, we can afford the occasional false positive.